contrib/clojure-package/test/good-test-ndarray.clj

```diff
@@ -22,15 +22,13 @@
 ;;
```

Comment:
This file should be excluded from the formatting - since the `test-write-to-file` uses it test whether the generator is working correctly and it makes the until test fail

Reply:
Thanks for your feedback. I will update it later tonight.